hedera-node/docs/design/services/smart-contract-service/accounts-blocklisting.md

```diff
@@ -0,0 +1,68 @@
+- As those default private test keys are well-known, anyone could have used them to create an account with the same alias but with different account admin key, leaving me with no control over the funds.
+
+In Hedera (desired behavior):
+- While executing tests against different environments, using some default private test keys (e.g. the ones configured with Hedera Local Node) to create and interact with accounts, I accidentally executed my tests on Mainnet and transferred some funds to an account with alias derived from one of the Hedera Local Node keys.
+- As those default private test keys are well-known, the ledger can be configured to own those EVM addresses and block any transfers to them, so that I cannot accidentally send funds to them.
+
+## Goals
+
+- We want to block a list of known addresses, so that no one can “squat” on them and exercise full control on any accidentally sent and blocked funds. Sending funds to such blocked addresses should not be successful as these accounts are considered “compromised”.
+
+## Non Goals
+
+- We don't consider unblocking already blocked accounts.
+- We don't consider what happens to a blocked account when it expires.
+
+## Architecture
+
+The implementation related to blocked accounts will be gated by a feature flag.
+
+### File containing the list of blocked accounts
+We will store the list of blocked accounts in a regular CSV resource file.
+Each row in the file will have a hex encoded ECDSA private key that corresponds to the EVM address that we want to block and an optional memo field that will be used to indicate the reason for blocking the account.
+
+### Creating the blocked accounts in state
+After the node starts (on handling the first transaction), for each of the EVM addresses that we want to block create a regular account with the following properties:
+- `receiverSigRequired` equal to `true`, thus blocking all future transfers to these accounts
+- `key` equal to the `GENESIS` key
+- `alias` equal to the EVM address derived from the corresponding private key
+- `memo` if specified for the blocked EVM address in the resource file, indicating the reason for blocking the account
+
+Anyone with access to the `GENESIS` key will have sole control on the accounts and the fact that a given alias can be associated only to a single account prevents squatting on the EVM addresses.
+The EVM already has checks integrated for accounts configured with `receiverSigRequired`.
+This approach does not require us to introduce a property file with random addresses and to have custom checks and logic executed in the EVM.
+
+### Externalizing the blocked accounts
+All created blocked accounts will be externalized as synthetic account creations.
+
+### New Classes
```

Comment:
Should add an acceptance tests section to highlight acceptance  test case cases for successful implementation

Reply:
Added.